classes/ConfigurationTest.php

```diff
@@ -264,83 +264,31 @@ public static function test_dir($relative_dir, $recursive = false, &$full_report
         return true;
     }
 
-    public static function test_file($file_relative)
+    public static function __callStatic($name, $arguments)
     {
-        $file = _PS_ROOT_DIR_.DIRECTORY_SEPARATOR.$file_relative;
+        static $recursiveTests = [
+            'test_img_dir',
+            'test_module_dir',
+            'test_mails_dir',
+            'test_translations_dir',
+            'test_config_sf2_dir'
+        ];
 
-        return file_exists($file) && is_writable($file);
+        return ConfigurationTest::test_dir($arguments[0], in_array($name, $recursiveTests));
```

Comment:
I'm not in favor of this kind of call, this make the project more difficult to understand. :/

Reply:
you're right, I've just reverted those changes. I would be create to cleanup a bit this file later though (some test_ functions are not used anymore)